docs/install/knative-with-operators.md

```diff
@@ -58,25 +58,25 @@ ko apply -f config/
 
 ### Verify the operator installation
 
-Verify the installation of Knative Serving Operator using the command:
+Verify the installation of Knative Operator using the command:
 
 ```
-kubectl get deployment knative-serving-operator
+kubectl get deployment knative-operator
 ```
 
 If the operator is installed correctly, the deployment should show a `Ready` status. Here is a sample output:
 
 ```
-NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
-knative-serving-operator   1/1     1            1           19h
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+knative-operator   1/1     1            1           19h
 ```
 
 ### Track the log
 
 Use the following command to track the log of the operator:
 
 ```
-kubectl logs -f $(kubectl get pods -l name=knative-serving-operator -o name)
+kubectl logs -f $(kubectl get pods -l name=knative-operator -o name)
```

Comment:
You should be able to use `kubectl logs -f deploy/knative-operator` here, I think.

Reply:
oh yes.